app/models/in_person_enrollment.rb

```diff
@@ -0,0 +1,22 @@
+class InPersonEnrollment < ApplicationRecord
+  belongs_to :user
+  belongs_to :profile
+  enum status: {
+    pending: 0,
+    passed: 1,
+    failed: 2,
+    expired: 3,
+    canceled: 4,
+  }
+
+  validate :profile_belongs_to_user
+
+  private
+
+  def profile_belongs_to_user
+    unless profile&.user == user
+      errors.add :profile, I18n.t('idv.failure.exceptions.internal_error'),
+                 type: :ipp_enrollment_mismatches_user
```

Comment:
```suggestion
                 type: :in_person_enrollment_user_mismatch
```
nit-picky, but I think this is clearer

Reply:
I disagree, but if you can point to a naming convention suggesting otherwise then I'll make the change.

Reply:
1. I think `ipp_enrollment_mismatches_user` makes it sound like the enrollment and the user don't match, but the problem is that the **profile** and the user don't match. Making it a more generic "user_mismatch" is more accurate without getting wordy, like "user_does_not_match_profile".
2. `in_person` is more readable than `ipp`; we're using them interchangeably in many places but want to move more towards `in_person`.

If you feel strongly about the `mismatches_user` part I'll defer, but I strongly prefer `in_person` to `ipp`.